ydb/_topic_writer/topic_writer_asyncio.py

```diff
@@ -107,11 +112,11 @@ async def write_with_ack_future(
         For wait with timeout use asyncio.wait_for.
         """
         input_single_message = not isinstance(messages, list)
-        if isinstance(messages, PublicMessage):
-            messages = [PublicMessage._create_message(messages)]
         if isinstance(messages, list):
             for index, m in enumerate(messages):
                 messages[index] = PublicMessage._create_message(m)
+        else:
+            messages = [PublicMessage._create_message(messages)]
```

Comment:
We can make it public in feature, if it will need.

Common scenario: writer.write("asd") or create own message: TopicMessage(data="123").
I don't see when customer want auto-convert data/message to TopicWriterMessage.